### ISSUE TITLE
Accessibility Audit: In High Contrast mode, dropdown on eligibility questionnaire not highlighting during error

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#932]
In a Macbook, turn on Increase Contrast (System Preferences -> Accessibility -> Display -> 'Increase Contrast Mode'). Then go to your Firefox browser and go to Settings -> General -> Language & Appearance -> Colors, and then click the 'Only with High Contrast themes' option on 'Override the colors specified by the page with your selections above'.

On a Windows machine, in Firefox, go to Settings -> Vision -> Use High Contrast and set 'Turn on high contrast' to On.

Now go to Firefox and go to the Eligibility Questionnaire. Go to the first question (Household Size). Without selecting a value in the dropdown, click the Next button. You will stay on the screen with nothing happening. What should happen is that the select element should be highlighted (this happens when running the browser normally).

NOTE: This is not replicable in Chrome.
